packages/experimental/NativeFontMetrics/src/NativeFontMetrics.types.ts

```diff
@@ -1,3 +1,9 @@
+export interface IFontMetrics {
```

Comment:
We try to stop using the "I" prefix in FURN now.

Reply:
Fixed.